Calculator/Calculator/Operator.swift

```diff
@@ -37,6 +37,8 @@ enum Operator: Character, CaseIterable, CalculateItem {
     }
     
     private func divide(_ lhs: Double, _ rhs: Double) -> Double {
+        guard lhs != 0.0, rhs != 0.0 else { return Double.nan }
+        
         return lhs / rhs
```

Comment:
이렇게 하면 1/0은 NaN이 되지만, 0/1도 NaN이 되버리네요! 
사실, 수학적으로는 1/0은 inf(무한대), 0/1은 0이 정확한 표현입니다 ㅎㅎ ;

제가 이전에 말씀드렸던 부분은 `Double`의 프로퍼티 중 하나인 `isInfinite`라는 프로퍼티입니다. 이 프로퍼티를 활용하면 값이 무한대인지 아닌지를 확인해볼 수 있습니다! 원래대로라면 inf혹은 ∞와 같이 표현하는 것이 정확하지만, 요구사항에서는 NaN이라 표시하라고 하였으니 이 프로퍼티를 활용하면 좀 더 정확히 표현이 가능하겠죠? 😁
```suggestion
        let result = lhs / rhs
        
        return result.isInfinite ? .nan : result
```

Reply:
조언 감사합니다 원비!